content/en/tracing/setup/php.md

```diff
@@ -232,6 +232,14 @@ DD_TRACE_DEBUG=true php -S localhost:8888
 
 #### URL Resource Names
 
+<div class="alert alert-warning">
+Enabling this feature <strong>could impact your billing</strong>.
+
+Increasing the number of unique resource names also increases cardinality which is a metric used in billing.
```

Comment:
It doesn't affect billing directly. We limit after a certain number of resources. I'd suggest dropping this off unless we are expecting any known repercussions. That is, had we auto-instrumented the framework, the same number of resources would have appeared as after normalization. Why do we need to warn them specifically? @labbati thoughts?

Reply:
This is good to know, so I would definitely rephrase it in a different and less scary way.

> had we auto-instrumented the framework, the same number of resources would have appeared as after normalization. Why do we need to warn them specifically?

When we support a framework we have access to the `route` name in addition to the url and we use the first one as a resource name.

So for example for a Symfony controller you set a route like this:

```
/** @Route("/users/{id}/permissions", name="users_permissions") */
public function getUserPermissions($id) {
   ....
}
```

So here for `/users/123/permissions` we have access to both `/user/{}/permissions` or `user_permissions` names and we can use the one that we like as the resource name. No cardinality issues in this case.

Reply:
I'll suggest handling it on per-use basis. Let's not use the warning banner to start with. Reason - it won't affect their billing, it'll affect the number of resources that they see after ballooning over a certain limit - which we can treat as a bug and increase the limit or look at the feature again. WDYT?